tvbwidgets/ui/ts_widget.py

```diff
@@ -277,6 +277,10 @@ def add_datatype(self, ts_tvb):
         self.logger.debug("Adding TVB TS for display...")
         self._populate_from_data_wrapper(data_wrapper)
 
+    def reset_data(self):
+        self.data = None
+        self.title_area.children = [self.instr_area]
```

Comment:
should we try to "close" previous children ? 
In theory it would be kinder to memory if we do something like bellow (did not try it here):
```
        for wid in self.title_area.children:
            wid.close()
```

Reply:
nevermind, this is not possible, as then we would spoil the `instr_area` also